test/unit/xiopak.unit.cpp

```diff
@@ -1,9 +1,21 @@
 #include "gtest/gtest.h"
 
 #include "cnglob.h"
+#include "cndefns.h"
```

Comment:
Is this necessary? Isn't it included in cnglob?

Reply:
You are right. I'll remove it.